interactive/api/api.py

```diff
@@ -103,9 +104,14 @@ class LoadResultsView(views.APIView):
     def get(self, request):
         logger.debug(
             f"load results | session key: {request.session.session_key}")
+        """
         response = controller.get_results_file(
             request.session.session_key).to_dict(
             orient='list')
+        """
+        partmc_response = pypartmc_controller.get_concentration(request.session.session_key)
+        response = {}
+        response['partmc'] = partmc_response
```

Comment:
This looks like it's specific to partmc. This needs to work for both partmc and music box. Would something like this work? Otherwise, you'll need to move all of this logic into `get_results_file`
```suggestion
        response = controller.get_results_file(
            request.session.session_key).to_dict(
            orient='list')
        partmc_response = pypartmc_controller.get_concentration(request.session.session_key)
        response['partmc'] = partmc_response
```